### PR TITLE
deploy.sh: do not load initial data during upgrades

### LINKED ISSUE #470: 'MultipleObjectsReturned' error during upgrade
```python
[cut]
[updated] Layer icebox_output_lossmap_644 (457/478)
[updated] Layer icebox_output_lossmap_645 (458/478)
Stopping process because --ignore-errors was not set and an error was found.
Exception: ('Failed to process event', MultipleObjectsReturned('get() returned more than one Link -- it returned 2!',))
```

### PR DESCRIPTION
**Pull request by daniviga**
This fixes #470. Installation tests are green: https://ci.openquake.org/job/zdevel_oq-platform/110/
Upgrade tests are done manually (and it worked).

### CODE REVIEW

File: openquakeplatform/bin/deploy.sh [hunk 1]
```diff
@@ -591,27 +590,24 @@ oq_platform_install () {
...
+            if function_exists "${app}_fixtureupdate"; then
+                "${app}_fixtureupdate" "$oqpdir"
+            fi
+        done
         openquakeplatform syncdb --all --noinput
     else
+        echo "WARNING: this operation could be destructive for the current version of oqplatform database schema, do proper backup before proceeding."
+        read -p "Open a new terminal, migrate the application database schema manually and then press <enter> to continue: " qvest
         openquakeplatform syncdb --noinput
     fi
```
Comment (nastasi-oq) [inline openquakeplatform/bin/deploy.sh]: Add a note about the **installation** and **updates** of **old** application and **new** application.

File: openquakeplatform/bin/deploy.sh [hunk 2]
```diff
@@ -664,6 +657,17 @@ oq_platform_install () {
...
     find oq-platform/openquakeplatform/migrations -type f \( -name "*.py" -or -name "*.sql" -or -name "*.sh" \) -exec cp "{}" "${MIGRATIONS_HISTORY}/" \;
 
+    if [ "$GEM_IS_INSTALL" == "y" ]; then
+        # updatelayers must be run again after the fixtures have been pushed
+        # to allow synchronization of keywords and metadata from GN to GS
+        openquakeplatform updatelayers
+    else
+        # the updatelayers is very expensive during upgrades due the number of layers
```
Comment (nastasi-oq) [inline openquakeplatform/bin/deploy.sh]: please, print with echo the comment to be more clear with user

Comment (daniviga) [inline openquakeplatform/bin/deploy.sh]: done

File: openquakeplatform/bin/deploy.sh (hunk 1 above)
Comment (daniviga) [inline openquakeplatform/bin/deploy.sh]: done

**Comment by nastasi-oq** [comment]
LGTM

**Comment by daniviga** [comment]
Closes #470.

**Pull request merged**